io/plugins/eu.esdihumboldt.hale.io.validation/plugin.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+   <extension
+         point="eu.esdihumboldt.hale.io.action">
+      <action
+            categoryName="Validator configurations"
+            id="eu.esdihumboldt.hale.io.validation.read"
+            name="Load validator configuration"
+            resourceName="validator configuration"
+            type="eu.esdihumboldt.hale.io.validation.ValidationConfigurationReaderImpl">
+      </action>
+      <advisor
+            action="eu.esdihumboldt.hale.io.validation.read"
+            class="eu.esdihumboldt.hale.io.validation.ValidatorConfigurationImportAdvisor"
+            id="eu.esdihumboldt.hale.io.validation.service">
+      </advisor>
+   </extension>
+   <extension
+         point="eu.esdihumboldt.util.resource">
+      <resourceType
+            id="validatorConfiguration"
+            name="Validator Configuration">
+      </resourceType>
+   </extension>
+   <extension
+         point="eu.esdihumboldt.hale.io.provider">
+      <provider
```

Comment:
Registering the reader for schematron I think would be better done in the schematron bundle. Like that schematron rules can only be imported if the corresponding validator is there.